src/tools/common/sss_tools.c

```diff
@@ -94,18 +93,25 @@ static errno_t sss_tool_confdb_init(TALLOC_CTX *mem_ctx,
     struct confdb_ctx *confdb;
     char *path;
     errno_t ret;
+    struct stat statbuf;
 
     path = talloc_asprintf(mem_ctx, "%s/%s", DB_PATH, CONFDB_FILE);
     if (path == NULL) {
         return ENOMEM;
     }
 
-    ret = confdb_setup(mem_ctx, path,
-                       SSSD_CONFIG_FILE, CONFDB_DEFAULT_CONFIG_DIR,
-                       NULL, false, &confdb);
+    ret = stat(path, &statbuf);
+    if (ret != 0) {
+        ret = errno;
+        DEBUG(SSSDBG_FATAL_FAILURE,
+              "Can't access '%s', probably SSSD isn't configured\n", path);
+        return ret;
+    }
+
+    ret = confdb_init(mem_ctx, &confdb, path);
     talloc_zfree(path);
     if (ret != EOK) {
-        DEBUG(SSSDBG_FATAL_FAILURE, "Unable to setup ConfDB [%d]: %s\n",
+        DEBUG(SSSDBG_FATAL_FAILURE, "Unable to connect to configDB [%d]: %s\n",
```

Comment:
I don't know what was your intention, but did you perhaps mean "config DB" (with space)?

Reply:
Fixed (and rebased).